UMLEditor/UMLEditor/Views/MainWindow.axaml.cs

```diff
@@ -977,31 +976,29 @@ private void Add_Method_OnCLick(object sender, RoutedEventArgs e)
                     // Taking the className string and accessing a corresponding class in active diagram
                     Class currentClass = _activeDiagram.GetClassByName(className);
                     
-                    // If the class does not exist...
```

Comment:
(Referring to line 952)
The name of the event handler is "Add_Method_OnCLick", the "L" in click should be lowercased